frontend/src/pages/PostDetailPage.tsx

```diff
@@ -0,0 +1,50 @@
+import { Navigate, useNavigate, useParams } from "react-router";
+
+import { PATHS } from "@/constants/paths";
+import { deletePost } from "@/features/post/api/deletePost";
+import { getPost } from "@/features/post/api/getPost";
+import PostDetail from "@/features/post/components/PostDetail";
+import Button from "@/shared/ui/Button";
+
+export default function PostDetailPage() {
+  const { postId } = useParams<{ postId: string }>();
+  const navigate = useNavigate();
+
+  const post = !postId ? null : getPost(postId);
```

Comment:
```suggestion
  const post = postId ? getPost(postId) : null ;
```

별건 아닌데 이게 훨씬 잘 읽히지 않나요? ㅎㅎ